src/pattern.rs

```diff
@@ -181,8 +306,45 @@ impl PatternLayout {
                     write!(w, "{}", thread::current().name().unwrap_or("<unnamed>"))
                 }
                 Chunk::Target => write!(w, "{}", target),
+                Chunk::Color(ref colorfmt) => {
+                    // Only deal with colors when logging to console
+                    if !console { continue }
+                    let mut w = term::stdout().unwrap();
```

Comment:
This unwrap seems concerning - it'll fire if stdout is redirected to a file iirc.